docs/output.md

```diff
@@ -40,12 +48,23 @@ The pipeline consists of the following steps:
 
 </details>
 
-The preprocessing step uses `pixelator single-cell amplicon` to create full-length amplicon sequences from both single-end and paired-end data.
-It returns a single fastq file per sample containing fixed length amplicons.
-This step will also calculate Q30 quality scores for different regions of the library.
-
 ### Quality control
 
+Quality control is performed using `pixelator single-cell preqc` and `pixelator single-cell adapterqc`.
+
+The preqc stage performs QC and quality filtering of the raw sequencing data.
+It also generates a QC report in HTML and JSON formats. It saves processed reads as well as reads that were
+discarded (i.e. were too short, had too many Ns, or too low quality, etc.). Internally `preqc`
+uses [Fastp](https://github.com/OpenGene/fastp), and `adapterqc`
+uses [Cutadapt](https://cutadapt.readthedocs.io/en/stable/).
+
+The `adapterqc` stage checks for the presence and correctness of the pixel binding sequences.
+It also generates a QC report in JSON format. It saves processed reads as well as discarded reads (i.e. reads that did not have a match for both pixel binding sequences).
+
+These processed and discarded FASTQ reads are intermediate and by default not placed in the output folder with the final files delivered to users.
+Set `--save_qc_passed_reads` and/or `--save_qc_passed_reads` to enable publishing of these files.
+Alternatively, set `--save_all` to keep all intermediary outputs of all steps.
+
 <details markdown="1">
 <summary>Output files</summary>
```

Comment:
HTML outputs are missing in the list of outputs

Reply:
Good catch, thanks! The html reports were missing from the output folder as well. Fixed now.